resources/styles/book-content/worked-examples.less

```diff
@@ -1,4 +1,6 @@
-.worked-example.note:first-child {
+// sometimes it's plural, sometimes not?
```

Comment:
Ok, that makes sense.  I meant to comment this and point you toward it to ask about and forgot to, glad you noticed.